src/aleph_client/__main__.py

```diff
@@ -57,7 +57,7 @@ def _load_account(
 
     if private_key_str:
         if not private_key_file:
-            return ETHAccount(private_key_str.encode())
+            return ETHAccount(private_key_str)
```

Comment:
Any particular reason for this change? Diving into the code, it seems like you can feed it bytes or strings, it doesn't really matter. I'd just like to understand why you want to change this.

Reply:
It was a confusing with another related bug that fixed after this one (https://github.com/aleph-im/aleph-client/pull/67)
but another behavior still there when we provide a private-key as text using the CLI

Reply:
OK, thanks !

Reply:
regarding the doc => https://eth-account.readthedocs.io/en/stable/eth_account.html#module-eth_account.account
the method accept only key as hex str or bytes not encoded string